components/mediation-ui/org.wso2.carbon.endpoint.ui/src/main/java/org/wso2/carbon/endpoint/ui/endpoints/failover/FailoverEndpoint.java

```diff
@@ -92,7 +104,14 @@ public void build(OMElement elem, boolean isAnonymous) {
         if (isAnonymous) {
             elem.addAttribute("name", "anonymous", elem.getOMFactory().createOMNamespace("", ""));
         }
-        org.apache.synapse.endpoints.Endpoint failoverEndpoint = EndpointFactory.getEndpointFromElement(elem, isAnonymous, new Properties());
+        org.apache.synapse.endpoints.Endpoint failoverEndpoint = EndpointFactory
+                .getEndpointFromElement(elem, isAnonymous, new Properties());
+
+        if (failoverEndpoint instanceof org.apache.synapse.endpoints.FailoverEndpoint) {
```

Comment:
This class also has the same name "FailoverEndpoint" as in  "org.apache.synapse.endpoints.FailoverEndpoint". So cannot import at the beginning.